source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -136,6 +136,22 @@
 
 /*-----------------------------------------------------------*/
 
+/*
+ * A pointer to the network interface is needed later when receiving packets.
+ */
+static NetworkInterface_t * pxMyInterface = NULL;
+
+static BaseType_t xNetworkInterfaceInitialise( NetworkInterface_t * pxInterface );
```

Comment:
Would you mind the change the names of these these static function to "prvSAM_", like e.g. 
`prvSAM_NetworkInterfaceInitialise()` ?
In the labs/IPv6 branch you will find man y examples of this naming.